brd-android/kyc/src/main/java/com/fabriik/kyc/ui/features/personalinformation/PersonalInformationViewModel.kt

```diff
@@ -104,4 +115,13 @@ class PersonalInformationViewModel(
                 && textValidator(name)
                 && textValidator(lastName)
     )
+
+    private fun isAgeValid(): Boolean {
+        val date = SimpleDateFormat(
+            KycApi.DATE_FORMAT,
+            Locale.getDefault()
+        ).format(currentState.dateOfBirth?.time!!)
+
+        return LocalDate.parse(date).plusYears(18).isBefore(LocalDate.now())
```

Comment:
Use Calendar or Date instead of LocalDate

Reply:
Should be fixed